spec/plane_spec.rb

```diff
@@ -16,15 +17,40 @@
 # Are you testing that?
 
 describe Plane do
+  let(:plane) {Plane.new}
+  let(:airport) {Airport.new}
+ 	end
 
-  xit 'is flying when created'
+  it 'is flying when created' do
+  	expect(plane.state).to eq 'Flying'
+  end
 
-  xit 'can land'
+  it 'is landed after landing' do
+  	expect(plane.state).to eq 'Landed'
+  end
 
-  xit 'is landed after landing'
+  it 'can take off if landed' do
+  	allow(airport.can_land).to receive(:false) {true}
+  	airport.land_plane(plane)
+
+  	allow(airport.can_take_off).to receive(:false) {true}
+  	expect(airport.can_take_off). to eq true
+  end
+
+  it 'is flying after take off' do
+  	allow(airport.can_land).to receive(:false) {true}
+  	airport.land_plane(plane)
+  	allow(airport.can_take_off).to receive(:false) {true}
+  	airport.take_off_plane(plane)
+  	expect(plane.flying).to eq 'Flying'
+	end
```

Comment:
Lint/BlockAlignment: `end` at 46, 1 is not aligned with `it 'is flying after take off' do` at 40, 2